src/components/Transactions/styles.ts

```diff
@@ -8,38 +9,82 @@ export const Row = styled.View`
   flex-direction: row;
   align-items: center;
   justify-content: flex-start;
-  border-top: 1px solid black;
-  border-bottom: 1px solid black;
+  height: 67px;
+  border: 0px solid ${Colors.lightGray};
+  border-bottom-width: 1px;
 `;
 
-export const DateIdContainer = styled.View`
-  flex: 4;
+export const DateContainer = styled.View`
+  flex: 1;
   flex-direction: column;
   justify-content: center;
   align-content: flex-start;
-  padding-left: 20px;
+  padding-left: 29px;
 `;
 
-export const CountContainer = styled.View`
+export const StatusContainer = styled.View`
   flex: 1;
   flex-direction: row;
   align-items: flex-end;
   justify-content: flex-end;
-  padding-right: 10px;
+  padding: 10px;
   margin-right: 30px;
 `;
 
-export const PriceContainer = styled.View`
-  flex: 2;
+export const ValueContainer = styled.View`
+  flex: 1;
   flex-direction: row;
-  align-items: flex-end;
-  justify-content: flex-end;
-  padding-right: 10px;
+  align-items: flex-start;
+  justify-content: flex-start;
+`;
+
+export const StatusComponentRed = styled.View`
+  text-align: center;
+  align-items: center;
+  border-radius: 9px;
+  width: 63px;
+  background: ${Colors.alertLightRed};
+  padding: 9px 9px;
+  border: 2px solid ${Colors.alertLightRed};
+`;
+
+export const StatusComponentGreen = styled.View`
+  text-align: center;
+  align-items: center;
+  border-radius: 9px;
+  width: 63px;
+  background: ${Colors.lightGreen};
+  padding: 9px 9px;
```

Comment:
Don't need to do `padding: 9px 9px`, can just do `padding: 9px` to specify all at once